src/pages/RadicalCollaboration/index.js

```diff
@@ -83,12 +30,95 @@ const RadicalCollaboration = () => {
+                <Typography variant="h6" color='primary' className={classes.subcardHeading}>Love the Index? Be an evangelist!</Typography>
+                <Typography className={classes.rightDtpLine1}>
+                 Help others and their communities discover, share and benefit from contributed projects on the index.
+                </Typography>
+                <Button component={Link} to='/support/share'  className={classes.buttonRight}>
+                 Share the CTI
+                </Button>
+                <br></br>
+              </Paper>
+            </Grid>
+            <Grid>
+              <Card className={classes.imgCardLeft}>
+                <CardMedia
+                  className={classes.cardMedia}
+                  image={'/images/Evangelize-Us.png'}
+                />
+              </Card>
+            </Grid>
+            <Grid item>
+              <Paper elevation={0} className={classes.innerVolunteerTextCardContainer}>
+                <Typography className={classes.cardHeading} variant='h4'>Volunteer With Us</Typography>
+                <Typography variant="h6" color='primary' className={classes.subcardHeading}>Help us improve the Civic Tech Index.</Typography>
+                <Typography className={classes.dtpLine1}>
+                 Please <a className={classes.inLineLinkText} href="https://www.hackforla.org/#about" target="_blank" rel="noopener noreferrer">complete this application</a>
+                 if you would like us to reach out to you directly or find us on our <a className={classes.inLineLinkText} href="https://www.hackforla.org/projects/civic-tech-index" target="_blank" rel="noopener noreferrer">project team page.</a>.
+                </Typography>
+                <Button href='https://www.hackforla.org/projects/civic-tech-index.html' color='textSecondary' className={classes.volButton}>
+                 Become a Volunteer
+                </Button>
+                <br></br>
+              </Paper>
+            </Grid>
+            <Grid>
+              <Card className={classes.imgCardVolunteer}>
+                <CardMedia
+                  className={classes.cardMedia}
+                  image={'/images/Volunteer-With-Us.png'}
+                />
+              </Card>
+            </Grid>
+            <Grid item>
+              <Paper elevation={0} className={classes.innerHelpTextCardContainer}>
+                <Typography className={classes.shareCardHeading} variant='h4'>Need Help?</Typography>
+                <Typography variant="h6" color='primary' className={classes.subcardHeading}>Can&apos;t find the answer youre looking for?</Typography>
+                <Typography className={classes.rightDtpLine1}>
+                   We&apos;ve shared some of our most frequently asked questions to help you out!  View our   <Link to="/faq"> FAQ  </Link>
+                  <span> to find answers or <a className={classes.inLineLinkText} href='/contact'>contact us.</a> </span>
```

Comment:
you can change this to material UI Link and it should always display the default color(blue). Do we need span?